projects/packages/sync/src/class-listener.php

```diff
@@ -417,8 +417,11 @@ public function get_actor( $current_filter, $args ) {
 		);
 
 		if ( $this->should_send_user_data_with_actor( $current_filter ) ) {
-			require_once JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php';
-			$actor['ip']         = jetpack_protect_get_ip();
+			if ( function_exists( 'jetpack_protect_get_ip' ) || ( defined( 'JETPACK__PLUGIN_DIR' ) && include_once JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php' ) ) {
```

Comment:
$ip= isset( $_SERVER['REMOTE_ADDR'] ) ? $_SERVER['REMOTE_ADDR'] : '';

It looks like we lost this line in the latest version to have the default value if jetpack_protect_get_ip is not available.

Reply:
Oopse! My mistake. Fixed.

Reply:
I'm afraid that this line looks too clever. Can we make it more obvious please? I would carry the defined check up and call the `include` conditionally only if it's set, then we can check whether the function exists. 
Otherwise this `if` takes a while to understand.